internal/pkg/cli/groups/cmd_groups.go

```diff
@@ -0,0 +1,13 @@
+// +build !windows
+
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package groups contains the names of command groups
+package groups
```

Comment:
nice! Can we rename the package to `group` and the file to `group.go`?   

The guideline is to not use plural in package names: https://rakyll.org/style-packages/